pkgs/applications/gis/tunnelx/default.nix

```diff
@@ -19,34 +19,35 @@ stdenv.mkDerivation (finalAttrs: {
   };
 
   nativeBuildInputs = [
-    makeWrapper
-  ];
-
-  buildInputs = [
     jdk
-  ];
-
-  runtimeInputs = [
-    survex
+    makeWrapper
   ];
 
   buildPhase = ''
+    runHook preBuild
+
     javac -d . src/*.java
+
+    runHook postBuild
   '';
 
   installPhase = ''
+    runHook preInstall
+
     mkdir -p $out/bin $out/java
     cp -r symbols Tunnel tutorials $out/java
     makeWrapper ${jre}/bin/java $out/bin/tunnelx \
       --add-flags "-cp $out/java Tunnel.MainBox" \
-      --set SURVEX_EXECUTABLE_DIR ${survex}/bin/ \
+      --set SURVEX_EXECUTABLE_DIR ${lib.getBin survex} \
```

Comment:
lib.getBin gets the bin output, so the /bin is still necessary

Reply:
Fixed